CHARTER.md

```diff
@@ -0,0 +1,117 @@
+
+   4. Participation in the Project through becoming a Contributor and Maintainer is open to anyone so long as they abide by the terms of this Charter. 
+
+   5. The TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of sub-projects, (2) set requirements for the promotion of Contributors to Maintainer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, and Maintainers, and create new roles, and publicly document any TSC roles, as it sees fit.
+
+   6. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and LF Decentralized Trust, a directed fund of The Linux Foundation.
+
+   7. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+
+      1. coordinating the technical direction of the Project;
+
+      2. approving project or system proposals (including, but not limited to, archiving and changing a sub-project’s scope);
+
+      3. organizing sub-projects and removing sub-projects;
+
+      4. creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+
+      5. appointing representatives to work with other open source or open standards communities;
+
+      6. establishing community norms, workflows, issuing releases, and security issue reporting policies;
+
+      7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+
+      8. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+      9. coordinating any marketing, events, or communications regarding the Project.
+
+3. **TSC Voting**
+
+   1. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+
+   2. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+
+   3. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
```

Comment:
where is section 7.c ? is this supposed to be 7.3 ?

Reply:
Yes, that's correct.  I'm not exactly sure what happened with markdown and the numbering system, so sorry about that.  The provision just sets a higher bar for charter changes and alternative licensing.